Exercise_A/about.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <a href = "mailto: abc@example.com">Send Email</a>
+</body>
+</html>
```

Comment:
Hello Nicole, 
Great work with this, though you could add your email address here for the example